.github/workflows/mkosi.yml

```diff
@@ -21,10 +21,11 @@ jobs:
           - debian
           - ubuntu
           - fedora
+          - opensuse
 
     steps:
     - uses: actions/checkout@v2
-    - uses: systemd/mkosi@v9
+    - uses: Werkov/mkosi@opensuse-github
```

Comment:
Is still change still relevant, since https://github.com/systemd/mkosi/pull/758 was merged?

Reply:
A change of this step is still relevant. The reference to v9 wouldn't work whereas pointing to the `main` branch would IMO be unstable (not reproducible test) and affecting multiple systemd PRs. The clearer solution to me seems to point the upcoming mkosi version tag.

 @DaanDeMeyer what are plans for tagging a new version of mkosi (for the reasons above)? (BTW thanks for merging systemd/mkosi#758.)

Reply:
A new release has been done, can you update this now?

Reply:
Done and thanks!